ddtelemetry-ffi/Cargo.toml

```diff
@@ -9,9 +9,10 @@ edition = "2021"
 [lib]
 # LTO is ignored if "lib" is added as crate type
 # cf. https://github.com/rust-lang/rust/issues/51009
-crate-type = ["staticlib", "cdylib"]
+crate-type = ["lib", "staticlib", "cdylib"]
```

Comment:
Why do you need lib? Note that due to the above comment, you will lose LTO and your library size will increase.

Reply:
This is a just a temporary hack to hijack the current build script and re-esxport ddtelemetry-ffi inside the profiler-ffi crate, to build and the telemetry example using the artifact built by the build-profiler-ffi script

Once I figure out how to actually build the library, I'll remove this